go/kbfs/libkbfs/leveldb.go

```diff
@@ -118,6 +123,41 @@ func openLevelDBWithOptions(stor storage.Storage, options *opt.Options) (
 	return &LevelDb{db, stor}, nil
 }
 
+var initialStartTime = time.Now()
+
+// openLevelDBStorage opens read-write leveldb storage based on a path.
+func openLevelDBStorage(
+	ctx context.Context, p string, log logger.Logger) (storage.Storage, error) {
+	// If we can't open the storage file because another process is
+	// still alive that has it open, wait a bit for that other process
+	// to be killed and release the resource.  Count the time from the
+	// beginning of the process's initialization, so we don't wait
+	// N*timeout for every DB we try to open.
+	eagainDeadline := initialStartTime.Add(leveldbMaxWaitOnEagain)
+	logged := false
+	for time.Now().Before(eagainDeadline) {
```

Comment:
Shouldn't this be using `config.Clock()`?

Reply:
Ideally yes, but we don't have a `config` here, and even if we plumbed it here we wouldn't have one when `initialStartTime` is set.  I guess I could do that in a `sync.Once` within this method but since this code doesn't get tested it didn't see, worth it.